modules/preview-react/pill/lib/usePillModel.tsx

```diff
@@ -11,10 +11,13 @@ export const usePillModel = createModelHook({
      * Use to disable a pill.
      */
     disabled: false,
+    id: '',
   },
 })(config => {
+  const id = useUniqueId();
   const state = {
     ...config,
+    id: config.id || id,
```

Comment:
`useUniqueId` takes an optional override.

```suggestion
  const id = useUniqueId(config.id);
  const state = {
    id,
    ...config,
```